spec.html

```diff
@@ -2053,7 +2053,8 @@ <h1>The Object Type</h1>
         </li>
       </ul>
       <p>Properties are identified using key values. A property key value is either an ECMAScript String value or a Symbol value. All String and Symbol values, including the empty String, are valid as property keys. A <dfn id="property-name">property name</dfn> is a property key that is a String value.</p>
-      <p>An <dfn id="integer-index">integer index</dfn> is a String-valued property key that is a canonical numeric String (see <emu-xref href="#sec-canonicalnumericindexstring"></emu-xref>) and whose numeric value is either *+0* or a positive integer &le; 2<sup>53</sup> - 1. An <dfn id="array-index">array index</dfn> is an integer index whose numeric value _i_ is in the range <emu-eqn>+0 &le; _i_ &lt; 2<sup>32</sup> - 1</emu-eqn>.</p>
+      <p>A <dfn id="uint53-string" oldids="integer-index">Uint53-string</dfn> is a String-valued property key that is a canonical numeric String (see <emu-xref href="#sec-canonicalnumericindexstring"></emu-xref>) and whose numeric value is either *+0* or a positive integer &le; 2<sup>53</sup> - 1. A <dfn id="uint32-string" oldids="array-index">Uint32-string</dfn> is a Uint53-string whose numeric value _i_ is in the range <emu-eqn>+0 &le; _i_ &lt; 2<sup>32</sup> - 1</emu-eqn>.</p>
```

Comment:
```suggestion
      <p>A <dfn id="uint53-string" oldids="integer-index">Uint53-string</dfn> is a String-valued property key that is a canonical numeric String (see <emu-xref href="#sec-canonicalnumericindexstring"></emu-xref>) and whose numeric value _i_ is an integer in the range <emu-eqn>+0 &le; _i_ &lt; 2<sup>53</sup> - 1. A <dfn id="uint32-string" oldids="array-index">Uint32-string</dfn> is a Uint53-string whose numeric value _i_ is in the range <emu-eqn>+0 &le; _i_ &lt; 2<sup>32</sup> - 1</emu-eqn>.</p>
```

Reply:
The change from "&le;" to "&lt;" excludes the value 2^53 - 1, which is currently included. I'm guessing you didn't intend that bit.

Reply:
Nope, just suggesting to use the same style of description as below.

Reply:
Well, that aside, the thing about the suggested change is that it would admit the value `"-0"` as a Uint53-string and Uint32-string (because `-0` is an integer and `+0 &le; -0` is true), which the current definition doesn't. So I think that would be backwards-incompatible.

Reply:
Maybe it's best to just specify in terms of mathematical values then?

Reply:
It might be, but this PR is just about replacing some terms, not about reformulating definitions.